spec/index.html

```diff
@@ -922,34 +922,34 @@ <h3>Literals</h3>
 
     <p>Literals are used for values such as strings, numbers, and dates.</p>
 
-    <p>A <dfn data-local-lt="RDF literal">literal</dfn> consists of
+    <p>A <dfn class="export" data-lt="RDF literal">literal</dfn> consists of
       two, three, or four components, as below:</p>
 
     <ol>
-      <li>A <dfn>lexical form</dfn>, being an <a data-lt="string">RDF string</a>.
-      <li>A <dfn>datatype IRI</dfn>, being an <a>IRI</a>
+      <li>A <dfn class="export">lexical form</dfn>, being an <a data-lt="string">RDF string</a>.
```

Comment:
Technically optional, added for consistency.
```suggestion
      <li>A <dfn class="export">lexical form</dfn>, being an <a data-lt="string">RDF string</a>.</li>
```